Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -11,12 +11,12 @@ struct DashboardView: View {
     @Environment(ActionService.self) var actionService
     @Environment(\.scenePhase) var scenePhase
     @State private var isShowingAddView = false
-    @Bindable private var viewModel: DashboardViewModel
+    @State private var viewModel: DashboardViewModel
     @State private var isShowingEditView = false
     @State private var selectedMaintenanceEvent: MaintenanceEvent?
     
     init(userUID: String?) {
-        viewModel = DashboardViewModel(userUID: userUID)
+        self._viewModel = State(initialValue: DashboardViewModel(userUID: userUID))
```

Comment:
```suggestion
        _viewModel = State(initialValue: DashboardViewModel(userUID: userUID))
```

The `self` isn't actually needed here

We could've also changed it to : `.init(initialValue: DashboardViewModel(userUID: userUID))` but this is slightly more confusing because you'd have to know `.init` meant you were initializing a `State`